### ISSUE TITLE
Path must be a string error

### ISSUE THREAD

**Issue by tnrich** [nspragg/filehound#23]
Hey there,

I'm getting the following error when using this package:

![image](https://cloud.githubusercontent.com/assets/2730609/15067125/fc869f56-1322-11e6-8ef6-60c2b59601ef.png)

I'm not quite sure why. You docs seem to suggest that paths can be an array, although there are no actual examples of this.

```
.paths(paths...) -> FileHound

Directories to search. Accepts one or more directories or a reference to an array of directories

Parameters

path - array of directories
Returns

Returns a FileHound instance
```

Any help would be appreciated. Thanks!

**Comment by nspragg** [comment]
@tnrich thanks for raising this issue. This should be fixed in v1.7.1 

You can now specify paths using an array:
**filehound.create().paths(myArrayOfPaths).find()**

or as variable arguments:
**filehound.create().paths(path1, path2)**

You can also call .path for a single directory:
**filehound.create().path(myPath).find()**

**Comment by tnrich** [comment]
Awesome, thanks @nspragg
